CONTRIBUTING.md

```diff
@@ -5,32 +5,32 @@
-Please read [OEO wiki](https://github.com/OpenEnergyPlatform/ontology/wiki/Best-Practice-Principles) carefully.
+Please read the [OEO best practices](https://github.com/OpenEnergyPlatform/ontology/wiki/Best-Practice-Principles) carefully.
 
 ### Workflow
 
-The development of a feature for this repository is inspired from the workflow described 
+The workflow for contributing to this project has been inspired by the workflow described 
 by [Vincent Driessen](https://nvie.com/posts/a-successful-git-branching-model/):
 
 1. Create
    [an issue](https://help.github.com/en/articles/creating-an-issue) on
-   the github repository
+   the GitHub repository describing the problem and proposed solution.
 
-    Choose the right issue for you among the choices you have on github:
+    Choose the right issue type from among the available choices:
 
-    A) Including a new term to the ontology. Only 1 new term (with a few subclasses, if at all)
+    A) Adding a new entity to the ontology. Ideally, each issue should correspond to only one new term (with only a few subclasses, if needed)
     
     B) Restructuring existing parts of the ontology
 
-    C) Updating definition of the ontology
+    C) Updating definitions of existing entities in the ontology
 
     D) Other issue
 
-    Discussion about the implementation details should occur within this issue. [Assign a project](https://help.github.com/en/github/managing-your-work-on-github/adding-issues-and-pull-requests-to-a-project-board#adding-issues-and-pull-requests-to-a-project-board-from-the-sidebar) (default "Issues") to the issue
+    [Assign a project](https://help.github.com/en/github/managing-your-work-on-github/adding-issues-and-pull-requests-to-a-project-board#adding-issues-and-pull-requests-to-a-project-board-from-the-sidebar) (default "Issues") to the issue
 
-2. Create a branch from `dev` to work on your issue (see below, the "Branch name convention" section)
+    Discussion about the implementation should take place within the issue. **Important**: Please discuss the proposal within the issue thread **before** starting to work on a solution. For minor changes, which include small changes to improve clarity of definitions and the addition of clarifying annotations, at least one other person from the project team should agree to the proposed change before it is implemented. For major changes, which include adding new entities and restructuring the ontology, at least two other members of the project team need to agree to the change before it is implemented, which should include at least one domain expert and at least one ontology expert. Issues which are contentious, for which it is difficult to reach agreement, should be added to the agenda of the next ontology working group teleconference for discussion to reach agreement amongst the full working group. Subsequent to such discussion, the issue should be updated with a documented record of the conclusions reached.   
```

Comment:
```suggestion
    Discussion about the implementation should take place within the issue. **Important**: Please discuss the proposal within the issue thread **before** starting to work on a solution. For minor changes, which include small changes to improve clarity of definitions and the addition of clarifying annotations, at least one other person from the project team should agree to the proposed change before it is implemented. For major changes, which include adding new entities and restructuring the ontology, at least two other members of the project team need to agree to the change before it is implemented, which should include at least one domain expert and at least one ontology expert. Issues which are contentious, for which it is difficult to reach agreement, should be added to the agenda of the next ontology working group teleconference for discussion to reach agreement amongst the full working group. Subsequent to such discussion, the issue's first thread should be updated with a documented record of the conclusions reached.   
```

Reply:
I just make one suggestion ("the issue's first thread" instead of "the issue" towards the end of the paragraph) but as the whole paragraph is only on one line the diff is large.